crates/aptos/src/account/balance.rs

```diff
@@ -88,10 +105,49 @@ impl CliCommand<Vec<AccountBalance>> for Balance {
             .parse::<u64>()
             .unwrap();
 
-        return Ok(vec![AccountBalance {
+        Ok(vec![AccountBalance {
             asset_type: "coin".to_string(),
             coin_type: Some(coin_type.to_string()),
             balance,
-        }]);
+        }])
+    }
+
+    async fn fa_balance(
+        self,
+        account: AccountAddress,
+        fa_addr: AccountAddress,
+    ) -> CliTypedResult<Vec<AccountBalance>> {
+        let client = self.rest_options.client(&self.profile_options)?;
+        let response = client
```

Comment:
can we use the new api?
same as the coin query.

Reply:
oh yeah, if your change is in then we can